docs/bin/plugin_formatter.py

```diff
@@ -132,6 +134,9 @@ def rst_xline(width, char="="):
     return char * width
 
 
+test_list = partial(is_sequence, include_strings=False)
```

Comment:
So while it's been said that this would only be python-centric code, as I suspected it is now trickling into the Jinja filter. Sequence, list, strings, the mess is even messier now :-/ Both in Ansible python and Jinja.

Reply:
It's just a helper function. So everyone reinventing it would not have to do this again.